packages/desktop-electron/index.js

```diff
@@ -95,8 +96,8 @@ function createBackgroundWindow(socketName) {
     title: 'Actual Server',
     webPreferences: {
       nodeIntegration: true,
-      contextIsolation: false
-    }
+      contextIsolation: false,
+    },
```

Comment:
Are these files enrolled in linting/formatting now?

Reply:
Maybe not for this PR, but I would like to lift the ESLint/Prettier config to the root of the project and run it on everything from there.

Reply:
It's my editor auto-applying the prettier config. Yes: I think we should enroll everything to eslint/prettier after this PR is merged. I was hesitant doing it before because I wanted to first upgrade eslint to v8 :)